### PR TITLE
[t140897][ADD] Created new module stock_quant_analytic

### PR DESCRIPTION
**Pull request by BT-atornero**
This module allows to add analytic accounts on stock moves using Inventory Adjustments.

 
<div> Links to Odoo: </div> <ul>
<li><a target="_blank" href="https://braintec.com/web#view_type=form&model=project.task&id=140897">[t140897] [NETA15] [DEV] Migration v15 - Missing OCA for Analytic Accounts on Inventory Adjustments</a></li>
</ul>
<div>Affected Modules:</div>
<table><thead><tr><th>Module</th><th>Ext</th></tr></thead>
<tr><td>stock_quant_analytic</td><td>.rst, .png, .html, .py, .xml</td></tr>
</tbody></table>

### CODE REVIEW

File: stock_quant_analytic/tests/test_stock_quant_analytic.py [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+
+class TestStockQuantAnalytic(TransactionCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        AccountAnalyticTag = cls.env["account.analytic.tag"]
+        ProductProduct = cls.env["product.product"]
+        eur = cls.env["res.currency"].search([("name", "=", "EUR")])
+        eur.active = True
```
Comment (BT-jmontero) [inline stock_quant_analytic/tests/test_stock_quant_analytic.py]: What is this for?

File: stock_quant_analytic/views/res_config_settings_views.xml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+        <field name="model">res.config.settings</field>
+        <field name="inherit_id" ref="stock.res_config_settings_view_form" />
+        <field name="arch" type="xml">
+            <div id="production_lot_info" position="after">
+                <h2>Analytic</h2>
+                <div class="row mt16 o_settings_container">
+                    <div
+                        class="col-12 o_setting_box"
+                        id="analytic_setting"
+                        groups="base.group_no_one"
```
Comment (BT-jmontero) [inline stock_quant_analytic/views/res_config_settings_views.xml]: Why this group here?

**Review by BT-jmontero — COMMENTED** [review]
LGTM! Though have a look at my few comments just in case it makes sense to change something

File: stock_quant_analytic/views/res_config_settings_views.xml (hunk 2 above)
Comment (BT-atornero) [reply stock_quant_analytic/views/res_config_settings_views.xml]: woops, copy paste mistake

File: stock_quant_analytic/tests/test_stock_quant_analytic.py (hunk 1 above)
Comment (BT-atornero) [reply stock_quant_analytic/tests/test_stock_quant_analytic.py]: I thought that would fix my tests when they werent working and I forgot to remove it

**Pull request merged**